scripts/01_DataExplorations.R

```diff
@@ -51,6 +51,7 @@ non_tens <- data %>%
   labs(title="Non-Multiples", col="") +
   ylab("Occurrences") + xlab("duration on log-axis")
 
+# TODO: Add explanation for legend, i.e. 0 is ..., 1 is ...? (It's clear from the script, but not in the plot as far as I can tell)
```

Comment:
Changed so that it includes both utterance- and word-initial data, and no more 0's and 1's